src/Components/Pages/duneBook.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getDumeBook } from '../../Redux/Books/reducerBooks';
+import banner from '../../Assets/Images/banner.webp';
+
+export default function DuneBook() {
+  const [filter, setFilter] = useState('');
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books);
+
+  useEffect(() => {
+    if (books.length === 0) dispatch(getDumeBook());
+  }, []);
+
+  const filterBook = books.filter((item) => {
+    const items = item.name.toLowerCase().includes(filter.toLowerCase());
+    return items;
+  });
+
+  return (
+    <>
+      <div className="search">
+        <input
+          type="text"
+          placeholder="Filter"
+          value={filter}
+          onChange={(e) => setFilter(e.target.value)}
+        />
+        <span className="material-symbols-outlined">search</span>
+      </div>
+      <div className="Banner">
+        <img src={banner} alt="Banner SectionImage" />
+        <h1 className="banner-title">Bob&apos;s Burgers Characters</h1>
+      </div>
+      <section>
+        {filterBook !== [] && filter === '' && (
+          books.map((book) => (
+            <article key={book.id}>
+              <div className="card-image">
+                <img src={book.image} alt="" width={80} />
+                <span className="material-symbols-outlined" id={book.id}>
+                  <Link to={`/characters/${book.id}`}>
+                    arrow_circle_right
+                  </Link>
+                </span>
+              </div>
+              <div className="card-body">
+                <h3 className="card-title">{book.name}</h3>
+                <p>{book.gender}</p>
+              </div>
+            </article>
+          ))
+        )}
+        {filterBook && filter !== '' && (
+          filterBook.map((book) => (
+            <article key={book.id}>
+              <div className="card-image">
+                <img src={book.image} alt="" width={80} />
+                <span className="material-symbols-outlined" id={1}>
+                  <Link to={`/characters/${book.id}/details`}>
+                    arrow_circle_right
+                  </Link>
```

Comment:
Kindly make sure that your cards have alternating background color shades as per the design specification.

![image](https://user-images.githubusercontent.com/38088818/195978941-e340cdb3-4b99-4d37-9ffa-2882a4c411dd.png)

Design specifications for card backgrounds:

![image](https://user-images.githubusercontent.com/38088818/195979036-653f67e7-ac07-4f8f-b5d6-44305df9f8c2.png)